doc/help/pypeit_obslog.rst

```diff
@@ -0,0 +1,85 @@
+.. code-block:: console
+
+    $ pypeit_obslog -h
+    usage: pypeit_obslog [-h] [-r ROOT] [-k] [-c COLUMNS] [-b] [-g] [-i] [-s SORT]
+                         [-e EXTENSION] [-d OUTPUT_PATH] [-o] [-f FILE]
+                         spec
+    
+    Construct an observing log for a set of files from the provided spectrograph
+    using PypeItMetaData.
+    
+    positional arguments:
+      spec                  A valid spectrograph identifier: gemini_flamingos1,
```

Comment:
spec is required here but optional for pypeit_setup. Symmetry between the arguments here is desirable, so we might consider making spec required for pypeit_setup. Not sure why it is not required.

Reply:
Yeah, I agree with you about symmetry.  spec is required by pypeit_setup, even if it isn't a positional argument.  I'm not sure what's better: (1) matching pypeit_obslog to pypeit_setup, meaning that people will always have to use the -s tag in the former, or (2) changing pypeit_setup, meaning that everyone that uses it now will have to change how they execute it.  My guess is the former, but let me know what you think.

Note that I changed pypeit_setup so that it no longer barfs if you don't use the -r tag.  The default is now just to use the current directory, just as in pypeit_obslog.

Reply:
I don't think it is a big deal to change pypeit_setup, so I would go with what you think is most elegant. I prefer to have required arguments not be keyword arguments, but rather be positional arguments.